### PR TITLE
Create paid-packs-process

### PR DESCRIPTION
**Pull request by mchasepan**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by mchasepan** [comment]
@ShahafBenYakir can you plz review today, I want to push live for a meeting on Tuesday. 
I did not add it to the nav intentionally since its not applicable to most partners at this time.

File: docs/partners/paid-packs-process.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## 3. Create Private Repository
+
+Paid packs are not stored in the XSOAR open source repository but instead in the publisher's own, private GitHub repo. We'll help you [set it up](/docs/packs/premium_packs). 
+
+## 4. Build Paid Pack(s)
+
+Build the content for the paid packs. 
+
+- If you're submitting a premium content pack, add the dashboards, playbooks, etc. as you would normally.
```
Comment (ShahafBenYakir) [inline docs/partners/paid-packs-process.md:24]:
```suggestion
- If you're submitting a premium content pack, add the dashboards, playbooks, etc. as you would normally, revisit the [Contributor Guidlines](https://xsoar.pan.dev/docs/contributing/contributing#contributor-guidelines) if needed.
```

File: docs/partners/marketplace.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ With the release of Cortex XSOAR v.6 in August 2020, we are pleased to announce
 
 [Cortex XSOAR Marketplace](/marketplace) is a digital storefront for discovering, exchanging and contributing security orchestration content centrally within the Cortex XSOAR platform. It enables users to install turnkey content packs to help solve any security use case with content contributed by SecOps experts and the world’s largest SOAR partner community, including you.
 
-Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, and how to best market it, and get a healthy share of the revenue. The customer gets premium content to solve their use case. It's a win-win.
+Additionally, Partners have the ability to monetize their content on the Marketplace. You decide the content, the price, and how to best market it, and get a healthy share of the revenue. ([Paid pack process](/docs/partners/paid-packs-process).) The customer gets premium content to solve their use case. It's a win-win.
```
Comment (ShahafBenYakir) [inline docs/partners/marketplace.md:10]: Isn't this a bit wierd like this? why do we need this link twice on this page?

**Review by ShahafBenYakir — APPROVED** [review]
See my comments

**Pull request merged**